design/MultipleQueues.md

```diff
@@ -0,0 +1,24 @@
+# Multiple Queues
+
+Queues are created at Device creation time, and are exposed via `sequence<Queue> Device.queues`.
+There are multiple types of queues, denoted by the `.type` bitfield on the Queue object.
+Queues can support `graphics` operations, `compute` operations, both, or neither.
+"Neither" Queues can only execute copy commands.
```

Comment:
wow, a graphics queue with no copy operations, fancy!

Reply:
In D3D12, graphics queues can have copy operations.  Video related queues cannot.

Reply:
Cool! Please let us know when there's documentation about this. I can't really integrate this knowledge until the interfaces have public documentation though.

Reply:
Unfortunately, the documenation for video queues are a ways away. 

To be clear, I am not proposing that we add video queues to WebGPU for MVP, or even V1.0.  But I would like for them to be easily added one day in the future.  Until then, we should avoid having verbiage that says "absence of all flags implies foo behavior" because we may add new flags in the future that do not apply to foo behavior.

If we feel that adding a flag for copy may make developers nervous about needing to write code that handles graphics queues with no copy operations, then we should consider enums that clearly state what operations are supported for the enum value.